apps/tailwind-components/components/form/FieldInput.vue

```diff
@@ -3,13 +3,16 @@ import type {
   InputString,
   InputTextArea,
   InputPlaceHolder,
+  InputListbox,
```

Comment:
unused import

Reply:
Ah, yeah. I forgot to define it in the `type instanceComponent = ...`